Kernel/Tasks/Process.h

```diff
@@ -935,6 +929,8 @@ class Process final
 
     SpinlockProtected<RefPtr<Custody>, LockRank::None> m_current_directory;
 
+    SpinlockProtected<u32, LockRank::None> m_thread_count { 0 };
```

Comment:
Moving this into a separate spin lock protected variable removes it from the Process protected data. Is this variable not security-sensitive? Was it a mistake to put it in the protected data to begin with?

Reply:
Probably yes - it's not security sensitive as far as I can tell - if you change it, would you get more privileges? probably not.